metasploit-framework.gemspec

```diff
@@ -83,8 +83,7 @@ Gem::Specification.new do |spec|
   # NTLM authentication
   spec.add_runtime_dependency 'rubyntlm'
   # Needed by anemone crawler
-  # Locked until build env can handle newer version due to native compile issue in 1.15.x
-  spec.add_runtime_dependency 'nokogiri', '~> 1.14.0'
+  spec.add_runtime_dependency 'nokogiri'
```

Comment:
Nokogiri v16 is required for Ruby 3.3 support

But it looks like this was pinned previously by https://github.com/rapid7/metasploit-framework/pull/18103 when there were issues with compiling the nightly omnibus installers with older. Will require manually rebuilding ommius to see if the libxml bump in nokogiri fixes this or introduces new complexities